magento/magento2
# Changes for shopby block on category is still present also mode is to just show static block

### Description
    Magento Versions 2.2.5
   The shop by category navigation is not shown - as there are no products shown in the category just a cms block.


### Fixed Issues (if relevant)

magento/magento2#7765
    


### Manual testing scenarios
  1 Go to catalog
    >>categories
    >>select category
    >>select display settings
    >>display mode set >> static block only
  2 >>select Design
    >> Layout Update XML
  3 remove the code referenceContainer name="catalog.leftnav" remove="true" if present
    left nav (shop by category) should not display on frontend.
